Sources/RediStack/ConnectionPool/RedisConnectionPool+Configuration.swift

```diff
@@ -134,7 +136,7 @@ extension RedisConnectionPool {
             self.minimumConnectionCount = minimumConnectionCount
             self.connectionRetryConfiguration = (
                 (initialConnectionBackoffDelay, connectionBackoffFactor),
-                connectionRetryTimeout ?? .milliseconds(10) // always default to a baseline 10ms
+                connectionRetryTimeout ?? Self.defaultConnectionRetryTimeout
```

Comment:
this is a breaking change. Explicitly passing nil currently leads to a timeout of 10ms. We would change that to 60sec. This can lead to unexpected behavior for adopters.